### ISSUE TITLE
Feature Request: Add a way to export PhysX meshes fitted similarly to the Triangle Mesh but compatible with dynamic PhysX entities

### ISSUE THREAD

**Issue by LB-ArturSiewniak** [o3de/o3de#3303]
**Is your feature request related to a problem? Please describe.**
Only Convex and Primitive PhysX Meshes are compatible with dynamic PhysX entities, which means there are no "precise" PhysX Mesh generation options for assets with concavities.

**Describe the solution you'd like**
An option of creating PhysX Meshes with collision well-fitted to a mesh with concavities or very closely resembling one created with Triangle Mesh, but usable with dynamic PhysX entities (e.g. PhysX Rigid Body with Kinematic property enabled.).

**Describe alternatives you've considered**
Allowing Triangle Mesh PhysX Meshes to be used with dynamic entities.

**Screenshots**
Comparison between three different PhysX Meshes on the same base .fbx:

Triangle Mesh:
![TriangleMesh](https://user-images.githubusercontent.com/86952082/129893807-307dfbec-9333-4eee-b209-1c32da387f29.PNG)

Convex with Decompose Meshes property enabled:
![ConvexDecomposeMeshes](https://user-images.githubusercontent.com/86952082/129893919-342e3ab8-3d6f-4bb5-98b8-92dc9db7117f.PNG)

Primitive with Decompose Meshes property enabled:
![PrimitiveDecomposeMeshes](https://user-images.githubusercontent.com/86952082/129893991-9af43023-1ce1-4329-9f96-bf3b255a19ce.PNG)

**Comment by Jackie9527** [comment]
I have a same issue #2994.

**Comment by greerdv** [comment]
Triangle meshes should already be compatible with kinematic rigid bodies (but not non-kinematic ones). If the components are not allowing you to do that, there is a bug.

For non-kinematic rigid bodies, you can use the convex decomposition to split the mesh into convex parts which approximate the original shape (see https://o3de.org/docs/user-guide/components/reference/physx/collider/#dynamic-body-with-collider-using-convex-decomposition https://o3de.org/docs/user-guide/assets/fbx-settings/physx-export/)

**Comment by LB-ArturSiewniak** [comment]
Due to the combination of the issue I reported here: https://github.com/o3de/o3de/issues/3322, my misunderstanding of Kinematic PhysX Rigid Body/Set Kinematic Target, and simply a badly written Script Canvas graph, I wrongly assumed that the Triangle Mesh + Kinematic Rigid Body were incompatible. After some more testing it appears that the Kinematic PhysX Rigid Body with Triangle Mesh assigned work as intended. Sorry for the trouble and thank you for response! Closing the Feature Request.
